papers/papers.inc

```diff
@@ -8,6 +8,7 @@ if (is_file("data.inc")) {
 
 function paper_page($title, $authors, $presented, $abstract, $files, $mon, $day, $year, $bibtex = "") {
     print("\n\n<h3>Title: $title");
+    date_default_timezone_set('America/Los_Angeler');
```

Comment:
but shouldn't we at least give it one that's a real timezone?  Also, UTC or death!

Reply:
Hah.  How did PHP not complain about that?

Is the timezone string for UTC `UTC`?

Reply:
i don't know; that surprised me.  I believe in PHP you can use 'UTC' as a timezone.  Apparently it won't complain either way :).

Reply:
Yes, it looks like it's `UTC`.  I'll update.